src/patterns/dosdp-patterns/OMIM_phenotypic_series

```diff
@@ -0,0 +1,36 @@
+pattern_name: OMIM_phenotypic_series
+
+description: >-
+
+This patterrn is meant to be used for OMIM phenotypic series (OMIMPS), which are represented as grouping classes in Mondo. Note: 
+- every instance of this metaclass should be equivalent to (via annotated xref) to something in OMIMPS namespace
+- it will never have an asserted causative gene as logical axiom (and no single causative gene in text def)
+- it must never be equivalent to an OMIM:nnnnnn (often redundant with the above rule)
+- it must have an acronym synonym, e.g. HPE
+- it must have two or more subclasses (direct or indirect) that are equivalent to OMIMs
+- the subclasses should (not must) have a logical def that uses the PS as a genus (see https://github.com/monarch-initiative/mondo/blob/master/src/patterns/dosdp-patterns/disease_series_by_gene.yaml)
+- the OMIM subclasses must have acronym synonyms that are the parent syn + number, e.g. HPE1, HPE2
+- the primary label for the children should also be parent + {"type"} + number
+- the first member will usually have the same number local ID as the PS
+- the first member in OMIM usually has documentation that is pertinent to the parent PS
+- the members may(?) generally share high semantic similarity
+
+Examples: 
+[holoprosencephaly](http://purl.obolibrary.org/obo/MONDO_0016296) [OMIMPS:236100](https://omim.org/phenotypicSeries/PS236100), '3-M syndrome'(http://purl.obolibrary.org/obo/MONDO_0007477) [OMIMPS:236100](https://omim.org/phenotypicSeries/PS273750). 
+
+classes:
```

Comment:
@matentzn there is no equiv axiom for this. Shall we just have a generic subClassOf? Or leave this off altogether? Don't want to break docs

Reply:
Formally these always have a NC-NC equiv axiom to the PS... but this is not encoded in the edit file (except as annotated xref). Hmmm..

Reply:
I think we are stretching the scope of DOSDP if we try to encode this kind of logic.. I would suggest we start now seriously with the design of a pattern formalism from which the other ones can be generated; there then sky is the limit. Right now I would advocate no logical axiom in the pattern.